root/instrument/List.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2018 Shamroy Pellew
+//
+// This file is part of MusicBrainz, the open internet music database,
+// and is licensed under the GPL version 2, or (at your option) any
+// later version: http://www.gnu.org/licenses/gpl-2.0.txt
+
+const React = require('react');
+const Layout = require('../layout');
+const Frag = require('../components/Frag');
+const EntityLink = require('../static/scripts/common/components/EntityLink');
+const {l, lp} = require('../static/scripts/common/i18n');
+
+const Instrument = (props) => {
+  const instrument = props.instrument;
+  return (
+    <li>
+      <EntityLink entity={instrument} />
+      {instrument.description
+        ? <Frag>
+          {' — '}
+          <span className="description"
+            dangerouslySetInnerHTML={{__html: l(instrument.description)}} />
```

Comment:
I don't think it will work, because i18n.js doesn't support domains other than `mb_server` right now. (The instrument descriptions are in the `instrument_descriptions` domain.) Also, it appears node-gettext doesn't handle domains correctly until version 2 (we're on version 1). I'll submit a PR to correct these issues.

Reply:
https://github.com/metabrainz/musicbrainz-server/pull/599